crankers/src/bin/main.rs

```diff
@@ -0,0 +1,120 @@
+use std::{collections::HashMap, path::PathBuf, time::Duration};
+
+use anyhow::Context;
+use clap::Parser;
+use jito_bytemuck::AccountDeserialize;
+use jito_vault_core::{vault::Vault, vault_operator_delegation::VaultOperatorDelegation};
+use jito_vault_cranker::{restaking_handler::RestakingHandler, vault_handler::VaultHandler};
+use log::info;
+use solana_rpc_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::{pubkey::Pubkey, signature::read_keypair_file};
+
+#[derive(Parser)]
+struct Args {
+    /// RPC URL for the cluster
+    #[arg(short, long, default_value = "https://api.devnet.solana.com")]
+    rpc_url: String,
+
+    /// Path to keypair used to pay
+    #[arg(short, long)]
+    keypair: PathBuf,
+
+    /// Vault program ID (Pubkey as base58 string)
+    #[arg(long, default_value = "Vau1t6sLNxnzB7ZDsef8TLbPLfyZMYXH8WTNqUdm9g8")]
+    vault_program_id: Pubkey,
+
+    /// Restaking program ID (Pubkey as base58 string)
+    #[arg(long, default_value = "RestkWeAVL8fRGgzhfeoqFhsqKRchg6aa1XrcH96z4Q")]
+    restaking_program_id: Pubkey,
+
+    /// Interval in seconds between cranking attempts (default: 5 minutes)
+    #[arg(long, default_value = "300")]
+    crank_interval: u64,
+
+    /// Priority fees (in microlamports per compute unit)
+    #[arg(long, default_value = "10000")]
+    priority_fees: u64,
+}
+
+#[tokio::main]
+async fn main() -> anyhow::Result<(), anyhow::Error> {
+    env_logger::Builder::from_env(env_logger::Env::default().default_filter_or("info")).init();
+
+    let args = Args::parse();
+    let rpc_client = RpcClient::new_with_timeout(args.rpc_url.clone(), Duration::from_secs(60));
+    let payer = read_keypair_file(args.keypair).expect("read keypair file");
```

Comment:
can have these unwrap/expects return anyhow! or whatever the erro macro is instead of unwrapping